src/particles/distribution/Waterbag.H

```diff
@@ -0,0 +1,125 @@
+/* Copyright 2021 Ji Qiang, Axel Huebl
+ *
+ * This file is part of ImpactX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef IMPACTX_DISTRIBUTION_WATERBAG
+#define IMPACTX_DISTRIBUTION_WATERBAG
+
+#include <AMReX_Random.H>
+#include <AMReX_REAL.H>
+
+#include <cmath>
+
+
+namespace impactx
+{
+namespace distribution
+{
+    struct Waterbag
+    {
+        /** A Waterbag distribution
+         *
+         * Return sampling from a 6D Waterbag distribution.
+         *
+         * @param sigx,sigy,sigt for zero correlation, these are the related
+         *                                   RMS sizes (length); ...
```

Comment:
Did we mean to fill the `...` with more info?
```suggestion
         * @param sigx,sigy,sigt for zero correlation, these are the related
         *                       RMS sizes (length)
```

Reply:
@cemitch99 @qianglbl this slipped through, please fix in a follow-up PR :)